Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticList.tsx

```diff
@@ -114,24 +129,24 @@ export const DiagnosticList: React.FC<IDiagnosticListProps> = ({ diagnosticItems
       maxWidth: 90,
       isResizable: true,
       data: 'string',
-      onRender: () => {
-        return (
-          <div data-is-focusable css={tableCell}>
-            <div aria-label={formatMessage(`Bot is {botName}`, { botName })} css={content} tabIndex={-1}>
-              {botName}
-            </div>
-          </div>
-        );
-      },
+      onRender: (item: IDiagnosticInfo) => <BotNameRender item={item} />,
```

Comment:
Why do we need a separate BotNameRender component. Cant we just render the bot name here